scripts/ios/release.sh

```diff
@@ -35,17 +35,6 @@ XCFRAME_ZIP_PATH=`echo "$OUTPUT_OF_BUILD" | cut -d ";" -f 2` || exit $?
 echo "🚢  CHECKSUM: $CHECKSUM"
 echo "🚢  XCFRAME_ZIP_PATH: $XCFRAME_ZIP_PATH"
 
-echo "🚢 ensuring Swift Sargon build for release"
```

Comment:
Moved into `release.sh` - my mistake here was that the `Packag.swift` was copied inside of the `release.sh` script, incorrectly containing `let useLocalFramework = true`.